helm/metacontroller/crds/metacontroller-crds-v1.yaml

```diff
@@ -0,0 +1,534 @@
+
```

Comment:
can't it include this file from `manifests/` directory ?

Reply:
Unfortunately not (or at least I've not found a way).